### ISSUE TITLE
Active buttons (btn-sm) shine through popover

### ISSUE THREAD

**Issue by lyind** [picturepan2/spectre#278]
Using spectre.css 0.4.0: Active buttons shine through opened popovers.

Popover closed:
![popover-button-shines-through-0](https://user-images.githubusercontent.com/18163495/30186651-b83a07c6-9426-11e7-8b71-c2087465a3e8.png)

Popover opened:
![popover-button-shines-through-1](https://user-images.githubusercontent.com/18163495/30186658-c4c00310-9426-11e7-90db-3ee892eaac12.png)

Code of the button which shines through:
`<button data-id="serviceDisableButton" class="btn btn-sm active">Off</button>`

**Comment by picturepan2** [comment]
@lyind Is it possible to share your code with me? I can't make it happen here.

**Comment by lyind** [comment]
@picturepan2 While preparing a test-case for you I noticed the problem was a z-index set on the popover element in my own code. That defeated the "z-index: 400" assigned to popover-container in spectre.

I learned about the concept of "Stacking Contexts" which is related to the z-index property:
[What No One Told You About Z-Index](https://philipwalton.com/articles/what-no-one-told-you-about-z-index/)

Thank you for your time.
